diff --git htdocs/files/test.mp4 htdocs/files/test.mp4
similarity index 86%
rename from htdocs/files/test.mp4 htdocs/files/test.mp4
rename to htdocs/files/test.mp4 htdocs/files/test.mp4
new file mode 100644
index 0000000..0577cc2
Binary files /dev/null and htdocs/files/test.mp4 differ

diff --git scripts/console scripts/console
new file mode 100644
index 0000000..19b446b
--- /dev/null
+++ b/application/classes/controller/a_nice_file.php	(working copy)
@@ -60,6 +58,7 @@
+			$x = 3;
 		}

-
 		//
 		// Comment
 		$this->whatever = new Test();
@@ -82,7 +78,6 @@
 		parent::before();
 	}

-
 	/**
 	 * A comment
 	 *
@@ -119,7 +114,6 @@
 		$this->content = 'great';
 	}

-
 	/**
 	 * Something else
 	 *

diff --git somefile.xml somefile.xml
old mode 100644
new mode 100755
index 7d522eb..b36bfac
